zcash_client_backend/src/data_api/wallet.rs

```diff
@@ -513,14 +516,16 @@ where
         .map_err(Error::from)
 }
 
-/// Construct, prove, and sign a transaction using the inputs supplied by the given proposal,
-/// and persist it to the wallet database.
+/// Construct, prove, and sign a transaction or series of transactions using the inputs supplied by
+/// the given proposal, and persist it to the wallet database.
 ///
-/// Returns the database identifier for the newly constructed transaction, or an error if
+/// Returns the database identifier for eacy newly constructed transaction, or an error if
 /// an error occurs in transaction construction, proving, or signing.
 ///
-/// Note: If the payment includes a recipient with an Orchard-only UA, this will attempt
-/// to fall back to the transparent receiver until full Orchard support is implemented.
+/// When evaluating multi-step proposals, only transparent outputs of any given step may be spent
+/// in later steps; attempting to spend a shielded note (including change) output by an earlier
+/// step is not supported, because the ultimate positions of those notes cannot be known and
```

Comment:
"the ultimate positions of those notes in the global commitment tree" (which requires mining, as opposed to their position in the prior transaction).